exporters.py

```diff
@@ -141,8 +141,7 @@ def prepare_concatenated(self, data_list, job_list):
             job = dask.delayed(self.save_to_disk)(map(operator.itemgetter(0), data_list), self.output_filename, self.format)
         else:
             concat_result = dask.delayed(pd.concat)(map(operator.itemgetter(0), data_list), ignore_index=True)
-            convert_columns_result = dask.delayed(RawExtractor.convert_columns_to_category)(concat_result)
-            job = dask.delayed(self.save_to_disk)(convert_columns_result, self.output_filename, self.format)
+            job = dask.delayed(self.save_to_disk)(concatenate, self.output_filename, self.format)
```

Comment:
I think this should be `concat_result` instead of `concatenate` right?

Reply:
Yes, thank you! Sorry, autocomplete tricked me there and I only tested the `prepare_separated` pathway :blush: 
I've pushed a fix now.